uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -0,0 +1,419 @@
+/* eslint-env amd */
+
+define([
+  'common/angular',
+  'common/moment',
+  'common/lodash',
+  'tasks-assignments/controllers/controllers',
+  'tasks-assignments/services/contact',
+  'tasks-assignments/services/file',
+  'tasks-assignments/services/dialog',
+  'tasks-assignments/services/document',
+  'common/services/notification'
+], function (angular, moment, _, controllers) {
+  'use strict';
+
+  controllers.controller('ModalDocumentCtrl', ModalDocumentCtrl);
+
+  ModalDocumentCtrl.$inject = [
```

Comment:
ModalDocumentCtrl.$inject = ['$scope', ...
  ..., 'notification'];

^ No need to break lines in this case.